insights/parsers/iris.py

```diff
@@ -5,9 +5,9 @@
 This parser reads the output of ``/usr/bin/iris list``.
 """
 
-from insights.core import CommandParser
+from insights.core import CommandParser, IniConfigFile
 from insights.core.exceptions import SkipComponent
-from insights.core.plugins import parser
+from insights import LogFileOutput, parser
```

Comment:
The `LogFileOutput` can be added to L8 too, and let's keep this line as it was.

The reason for this is to keep the alpha order the importing.